Gibe.Navigation.Umbraco/Filters/NotRedirectAndNoTemplateFilter.cs

```diff
@@ -1,8 +1,6 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
 using Umbraco.Core.Models;
 
 namespace Gibe.Navigation.Umbraco.Filters
```

Comment:
Negative assertions always throw me. "NotRedirectAndNoTemplate" sounds like the opposite of what this actually does to me. "TemplateOrRedirect" could be the positive rewording and may be clearer?